sdk/ml/azure-ai-ml/azure/ai/ml/_schema/automl/nlp_vertical/nlp_fixed_parameters.py

```diff
@@ -6,7 +6,7 @@
 
 from marshmallow import fields, post_load
 
-from azure.ai.ml._restclient.v2022_10_01_preview.models import NlpLearningRateScheduler
+from azure.ai.ml._restclient.v2022_12_01_preview.models import NlpLearningRateScheduler
```

Comment:
does sweep job is managed by some other team? I saw some missing updates for sweep

Reply:
can you please let me know what are the updates that we have missed for sweep? thank you!!

Reply:
@wangchao1230 perhaps best for you/Doris as the original modifier to follow up with the other job types for their missed updates? and we can focus on automl in this PR

Reply:
Sure. Let's focus on automl in this PR.